src/main/resources/application.conf

```diff
@@ -0,0 +1,2 @@
+foo = "hello"
```

Comment:
Notice that you should be able also to have the env variable override in the same file as the default values! so application.conf would look like

`foo = "hello"
foo = ${?FOO}`

^ will override foo with env variable only if FOO is defined